templates/partials/footer.html

```diff
@@ -1,13 +1,32 @@
-<div class="row">
-    <div class="container">
-        <div class="col-12">
-            <div class="row footer">
-                <div class="col-lg-4"></div>
-                <div class="col-lg-4 text-center">
-                    <p><a href="icon-credit-source.html" target="_blank">Image References</a></p>
-                </div>
-                <div class="col-lg-4"></div>
-            </div>
+{% 
+    set footer_content = [
+    {"image": "images/logos/funders/reverse/logo-eccc-en.png", "link": "https://www.canada.ca/en/environment-climate-change.html"},
+    {"image": "images/logos/funders/reverse/canadian-foundation-innovation-logo-new-light.svg", "link": "https://www.innovation.ca/"},
+    {"image": "images/logos/funders/reverse/ontario-provincial-new-seeklogo 1.svg", "link": "https://www.canada.ca/en/environment-climate-change.html"},
+    {"image": "images/logos/funders/reverse/BritishColumbia_Logo_Horizontal_Grey_Reverse.png", "link": "https://www2.gov.bc.ca/gov/content/home"},
+    {"image": "images/logos/funders/reverse/economie-science-et-innovation-quebec-seeklogo 1.svg", "link": "https://www.economie.gouv.qc.ca/"},
+    {"image": "images/logos/funders/reverse/UofT-white-on-transparent.png", "link": "https://www.utoronto.ca/"},
+    {"image": "images/logos/funders/reverse/concordia-logo.svg", "link": "https://www.concordia.ca/"},
+    {"image": "images/logos/funders/reverse/mcgill-university-2 1.svg", "link": "https://www.mcgill.ca/"},
+    {"image": "images/logos/funders/reverse/ouranos_logo_horizontal_blanc.png", "link": "https://www.ouranos.ca/en"},
+    {"image": "images/logos/funders/reverse/computer-research-institute-of-montreal-crim-vector-logo 1.svg", "link": "https://www.crim.ca/en/"},
+    {"image": "images/logos/funders/reverse/logo-pcic.png", "link": "https://www.pacificclimate.org/"}
+] 
+%}
+
+
+ <div class="container-fluid footer">
+    <div class="row d-flex flex-sm-column flex-md-row">
+        <div class="align-items-center">
```

Comment:
this class should be on the parent div (it isn't doing anything at the moment)

Reply:
Changed to the parent and changed to justify-content-centre.